src/net_stat.cc

```diff
@@ -133,9 +132,12 @@ void parse_net_stat_arg(struct text_object *obj, const char *arg,
 
   if (arg == nullptr) { arg = DEFAULTNETDEV; }
 
+#if defined(__linux__)
```

Comment:
Move this `#if` so that it _only_ wraps the call to `update_gateway_info()`.

Reply:
Fixed now.